sdk/simulation/uinst/Makefile

```diff
@@ -63,7 +63,7 @@ enclave_mngr.o: enclave_mngr.cpp
 # since the '_SE3' function has assumptions on stack layout.
 #
 u_instructions.o: u_instructions.cpp
-	$(CXX) $(CXXFLAGS) $(CPPFLAGS) -O0 -Wno-error=cpp -c $< -o $@
+	$(CXX) $(CXXFLAGS) $(CPPFLAGS) -Wno-error=cpp -c $< -o $@
```

Comment:
What is the reason the modification is different from sdk/simulation/tinst/Makefile?

Reply:
I removed -O0 from both. The makefiles don't look the same, so the implementation of removing -O0 looks different.